x/crosschain/keeper/evm_hooks.go

```diff
@@ -306,6 +309,9 @@ func ParseZetaSentEvent(log ethtypes.Log, connectorZEVM ethcommon.Address) (*con
 	if err != nil {
 		return nil, err
 	}
+	if len(log.Topics) == 0 {
```

Comment:
For ZRC20 `ZetaSent` event, the number of topics would be `3` (signature + zetaTxSenderAddress + destinationChainId)
https://github.com/code-423n4/2023-11-zetachain/blob/b237708ed5e86f12c4bddabddfd42f001e81941a/repos/protocol-contracts/contracts/zevm/ZetaConnectorZEVM.sol#L67C20-L67C20

Reply:
The fix is just to prevent the panic, I think we already check for the format of the event